onprc_billing/resources/queries/onprc_billing/publicInvoicedItems.sql

```diff
@@ -12,51 +12,58 @@
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
+ *Update 2022.05.03 change of source 3rd try
  */
-SELECT
-  i.rowId,
-  i.invoiceId,
-  i.transactionNumber,
-  i.date,
-  i.invoiceDate,
-  i.Id,
-  i.item,
-  i.itemCode,
-  i.category,
-  i.servicecenter,
-  i.project,
-  i.debitedaccount,
-  i.creditedaccount,
-  i.faid,
-  i.investigatorId,
-  i.firstName,
-  i.lastName,
-  i.department,
-  i.mailcode,
-  i.contactPhone,
-  i.chargeId,
-  i.objectid,
-  i.quantity,
-  i.unitCost,
-  i.totalcost,
-  i.chargeCategory
 
-FROM onprc_billing.invoicedItems i
-WHERE ((SELECT max(rowid) as expr FROM onprc_billing_public.publicdataAccess da WHERE isMemberOf(da.userid) AND (
-    da.allData = true OR
-    (da.project = i.project) OR
-    --TODO: this needs to get cleaned up
-    (
-      da.investigatorId = i.investigatorId
-      OR da.investigatorId = i.debitedaccount.investigatorId
-      OR da.investigatorId = i.project.investigatorId
-    )
-  )) IS NOT NULL OR
 
-  --include if the user is either the project's PI, the account PI, or the financial analyst
-  isMemberOf(i.project.investigatorId.userid) OR isMemberOf(i.debitedaccount.investigatorId.userid) OR isMemberOf(i.project.investigatorId.financialAnalyst)
+SELECT
+    i.rowId,
+    i.invoiceId,
+    i.transactionNumber,
+    i.date,
+    i.invoiceDate,
+    i.Id,
+    i.item,
+    i.itemCode,
+    i.category,
+    i.servicecenter,
+    i.project,
+    i.debitedaccount,
+    i.creditedaccount,
+    i.faid,
+    i.investigatorId,
+    i.firstName,
+    i.lastName,
+    i.department,
+    i.mailcode,
+    i.contactPhone,
+    i.chargeId,
+    i.objectid,
+    i.quantity,
+    i.unitCost,
+    i.totalcost,
+    i.chargeCategory,
+    pr.investigatorid as Inv1,
+    pr.project as pr1
```

Comment:
You can remove these fields if you want.  This was just part of our troubleshooting.